packages/components/modal/src/close_modal.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from "@buoysoftware/anchor-layout";
+
+interface OwnProps {
+  closeModal?: (e: React.MouseEvent | React.KeyboardEvent) => void;
+}
+
+type CloseModalProps = OwnProps;
+
+export const CloseModal: React.FC<CloseModalProps> = ({
+  closeModal,
+}): React.ReactElement => (
+  <Box
+    data-testid="close-modal-button"
+    height="26px"
+    mr="-5.5px"
+    onClick={closeModal}
+    position="relative"
+    style={{ cursor: "pointer" }}
+    width="26px"
+  >
+    <Box
+      bg="grey50"
+      height="1.5px"
+      left="5.5px"
+      position="absolute"
+      style={{ rotate: "45deg" }}
```

Comment:
I wasn't sure if it was worth adding this as a custom style prop on `Box`, but I am open to the idea.